x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java

```diff
@@ -440,6 +441,19 @@ public void testFreezeAction() throws Exception {
         });
     }
 
+    public void testSetPriority() throws Exception {
+        createIndexWithSettings(index, Settings.builder().put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
+            .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 0).put(IndexMetaData.INDEX_PRIORITY_SETTING.getKey(), 100));
+        int priority = randomIntBetween(0, 99);
+        createNewSingletonPolicy("warm", new SetPriorityAction(priority));
+        updatePolicy(index, policy);
+        assertBusy(() -> {
+            Map<String, Object> settings = getOnlyIndexSettings(index);
+            assertThat(getStepKeyForIndex(index), equalTo(TerminalPolicyStep.KEY));
+            assertThat(settings.get(IndexMetaData.INDEX_PRIORITY_SETTING.getKey()), equalTo(String.valueOf(priority)));
```

Comment:
I think it may be simpler to use:

```suggestion
            assertThat(IndexMetaData.INDEX_PRIORITY_SETTING.get(settings), equalTo(priority));
```

Reply:
in this context, settings is `Map`, not a `Settings` object.

Reply:
Whoops good call, I'm so used to everything being a `Settings` object :)